docs/docs/advanced_usage.md

```diff
@@ -258,6 +258,30 @@ If we describe option `Usage: lets run --stg` we will actually get two env varia
 
 You can learn more about options in [Options section](config.md#options)
 
+### Command templates
+U can make command templates using .yaml features, for generic options, like
```

Comment:
Could you replace `U` with `you` please?

Reply:
ohhh... sure